global.R

```diff
@@ -0,0 +1,43 @@
+###### MEtaInsight ######
+
+library(BUGSnet)
+library(cowplot)
+library(data.table)
+library(dplyr)
+library(knitr)
+library(gemtc)
+library(ggiraphExtra)
+library(ggplot2)
+library(ggrepel)
+library(magick)
+library(metafor)
+library(netmeta)
+library(patchwork)
+library(plotly)
+library(plyr)
+library(shiny) 
+library(shinyalert)
+library(shinyAce)
+library(shinyBS)
+library(shinycssloaders)
+library(shinydashboard)
+library(shinyjs)
+library(shinyWidgets)
+library(stringr)
+library(rmarkdown)
+library(tidyr)
+
+
+#Call required library for pie plotting
+libplotrix <- function(){
+  install.packages('plotrix')
+  require(plotrix)
+} 
+ifelse(any(grepl('plotrix',installed.packages()[,1])), require(plotrix), libplotrix())
```

Comment:
These library loads can be refactored to a reusable function:
```
load_library <- function(library_name){
  if(!require(library_name, character.only = T)){
    install.packages(library_name)
  }
  require(library_name, character.only = T)
}
```

And then called by:
```load_library("plotrix")```

However this can be done in another feature / PR